packages/extension-ui/src/components/RadioGroup.tsx

```diff
@@ -28,14 +36,55 @@ const RadioGroup: React.FC<Props> = ({ className, defaultSelectedValue, onSelect
     [onSelectionChange]
   );
 
+  console.log('options', options);
+
+  const { alephOptions, otherOptions } = options.reduce(
+    (acc: { alephOptions: Option[]; otherOptions: Option[] }, option) => {
```

Comment:
When typing `reduce()`, it's better to put the `acc` type in the generic:
```ts
options.reduce<{ alephOptions: Option[]; otherOptions: Option[] }>((acc, option) =>
```
as it applies to the entire _reduce_, not just the a param of the callback function, thus e.g. making sure the second parameter to `reduce()` adheres. Otherwise the type is inferred from the array itself, which, if you're explicitly typing the params, I understand is not what you want.